pkgs/development/python-modules/generic/default.nix

```diff
@@ -45,6 +45,8 @@
 # Additional flags to pass to "pip install".
 , installFlags ? []
 
+, format ? "setup"
```

Comment:
If there are only two formats, perhaps calling the option explicitly "useWheel" or similar would be better.

Alternatively, you could just see if the name of the source ends with `.whl` (we're already doing something similar to detect if the source is a `.zip`)